dev-tools/mage/dockerbuilder.go

```diff
@@ -58,6 +59,11 @@ func newDockerBuilder(spec PackageSpec) (*dockerBuilder, error) {
 }
 
 func (b *dockerBuilder) Build() error {
+	variants := []string{""}
+	if os.Getenv("OFFLINE_DOCKER_IMAGE") == "true" || b.ServiceName == "elastic-agent" {
```

Comment:
We should document this somewhere in the Beats repo?

Reply:
Is there a place we already document similar things? It's my understanding that we don't have such a place. I'd be glad to document it, my only question is where?

Reply:
I started to put things into elastic agent readme: https://github.com/elastic/beats/tree/master/x-pack/elastic-agent Lets put it there so we have it and can figure out a better place later.

Reply:
I've since removed this and replaced it in the package spec, where it has an explanatory comment. I've also added a note to the README you mentioned about the behavior of the package spec.